Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift

```diff
@@ -9,7 +9,35 @@ import UIKit
 /// A view representing a switch item.
 /// :nodoc:
 public final class FormToggleItemView: FormValueItemView<Bool, FormToggleItemStyle, FormToggleItem> {
-    
+
+    private enum Layout {
+        static let switchWidth: CGFloat = 40.0
```

Comment:
the width is assumed, maybe its not 40 on iPad, or it will change in later iOS versions